docs/assets/extra.css

```diff
@@ -0,0 +1,62 @@
+:root {
+  --md-primary-fg-color: #084298; /* PrairieLearn blue */
+  --md-accent-fg-color: #106eb0; /* arbitrary darker blue */
+}
+
+/* Custom CSS for the link color */
+[data-md-color-scheme='default'],
+[data-md-color-scheme='full-width'] {
+  --md-typeset-a-color: #127ac4; /* arbitrary blue */
+}
+/* Allow captioned code blocks to function */
+code {
+  text-align: left;
+}
+
+/* Prevent wrapping attributes inside tables */
+td code {
+  white-space: nowrap;
+}
+
+.md-nav__item--section > .md-nav__link[for],
+.md-nav__title {
+  text-decoration: underline;
+}
```

Comment:
I don't love this; an underline reads like a link to me, but these aren't clickable on desktop displays.

Reply:
Not sure what is good

Reply:
IMO all caps is better! Any thoughts about how it would look if we also made them the primary color?

Reply:
I think it looks pretty good.